include/openthread/platform/crypto.h

```diff
@@ -174,6 +177,19 @@ typedef struct otPlatCryptoEcdsaKeyPair
     uint8_t mDerLength;
 } otPlatCryptoEcdsaKeyPair;
 
+/**
+ * @struct otPlatCryptoEcdsaKeyRef
+ *
+ * This structure represents an ECDSA key pair as a PSA keyRef.
+ *
+ * This structure is only used by OT core when `OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE` is enabled.
+ *
+ */
+typedef struct otPlatCryptoEcdsaKeyRef
+{
+    uint32_t mKeyRef;
```

Comment:
Use `otCryptoKeyRef` instead of `uint32_t`?

Reply:
Will push a fix in next commit.